controllers/events_controller.js

```diff
@@ -4,7 +4,7 @@ const worker = require("../config/Scheduler/worker");
 const ObjectId = require("mongoose").Types.ObjectId;
 const { AVATAR_URL } = require("../config/index");
 // import http status codes
-const { BAD_REQUEST } = require("../utility/statusCodes");
+const { BAD_REQUEST, FORBIDDEN } = require("../utility/statusCodes");
 // import constants
 const { USER_HASH_LENGTH, EVENT_HASH_LENGTH } = require("../config/index");
 // import helper functions
```

Comment:
setToken, checkToken not imported??

Reply:
I must have left it by mistake. Will change it!